doc/samples/readme-encode_linux.md

```diff
@@ -107,6 +107,7 @@ The following command-line switches are optional:
   | [-num_slice] | number of slices in each video frame. 0 by default. If num_slice equals zero, the encoder may choose any slice partitioning allowed by the codec standard.|
    |[-mss] |maximum slice size in bytes. Supported only with -hw and h264 codec. This option is not compatible with -num_slice option.|
   | [-mfs]|maximum frame size in bytes. Supported only with h264 and hevc codec for VBR mode.|
+  | [-BitrateLimit:<on,off>] | Turn this flag OFF to remove bitrate limitations imposed by the SDK encoder. Setting this flag OFF may lead to violation of HRD conformance and severe visual artifacts. The default value is ON, i.e. bitrate is limited.|
```

Comment:
These won't be artifacts, but the result of the requested bitrate, i.e. quality might be low.

Actually I disagree with the selected default value for this parameter. Following bitrate limit is needed for the small fraction of use cases, for example, encoding for DVD. Majority of the people don't need this. Moreover, they don't get it for libx264 which gives only a warning, but allows to encode with the selected bitrate. As a result when we have bitrate limit defaulted to ON we face the situation that people can't compare our encoder with someone else encoder apple to apple at least out of the box.

Summarizing:
1. I would like to see this feature be OFF by default at least on samples level. This implies both to sample encode and SMT.
2. I think we should not write about artifacts, but warn that there could be low quality if bitrate will be selected lower than required by profile/level/etc. combination. In which case it is recommended to enable this feature and see the difference.